client/src/pages/GroupsList.jsx

```diff
@@ -39,7 +39,6 @@ const GroupsList = () => {
 
     return (
         <>
-        <h2>Groups</h2>
         <FilterSlider onFilterChange={(status) => {setStatusFilter(status)}}/>
```

Comment:
We may benefit from a ListWrapper component since GroupsList and EventsList have some similarities